src/Termonad/Term.hs

```diff
@@ -323,8 +334,12 @@ createTerm handleKeyPress mvarTMState = do
       let newTabs = appendFL tabs notebookTab
           newTMState =
             tmState & lensTMStateNotebook . lensTMNotebookTabs .~ newTabs
+          -- TODO: I'm not 100% sure this is the right place to hook setShowTabs
+          -- into tab creation. If it is, I suppose setCurrPageAction should be
+          -- renamed to something less specific. mvarReturnAction?
```

Comment:
I don't have time this week to really take a good look at this, but I can go ahead and merge this PR if it currently works.  Then I can go back in a week or so and take a good look at it.